### ISSUE TITLE
Add how to handle with SSL certificate in SerenityRest

### ISSUE THREAD

**Issue by szanelato** [serenity-bdd/the-serenity-book#45]
Hi @wakaleo 

I was looking for a Serenity configuration for disable SSL certificates validation using SerenityRest (rest assured).

I'm facing the same error related here: https://github.com/rest-assured/rest-assured/issues/967 . I'm trying at first to disable the SSL verification. The approach that suggested at the related forum not gonna works fine for me because I have a complex environment running my tests. 

In short I was trying to figure out how to disable SSL verification for rest API tests using Serenity, but I didn't find anything at serenity documentation.

Is there anyway to do this in serenity.conf or apply others configurations?

Thanks.

**Comment by lukago** [comment]
Same issue here

**Comment by szanelato** [comment]
@lukago acctually I found a solution.
Try to add the annotation `.relaxedHTTPSValidation()` in the requests, in my case I'm using it like this:

public void myMethod() {
SerenityRest.given()
                .relaxedHTTPSValidation()
                .headers(headers)
                .when()
                .post(myEndpoint);
}

Doing this you will ignore those SSL certificate validations.

**Comment by Julio-betancur** [comment]
> @lukago acctually I found a solution. Try to add the annotation `.relaxedHTTPSValidation()` in the requests, in my case I'm using it like this:
> 
> public void myMethod() { SerenityRest.given() .relaxedHTTPSValidation() .headers(headers) .when() .post(myEndpoint); }
> 
> Doing this you will ignore those SSL certificate validations.

I love you
